src/modules/fhir/base.ts

```diff
@@ -0,0 +1,32 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { z } from 'zod'
+
+export const FHIRCodableConceptValidationSchema = z.object({
```

Comment:
I wouldn't be surprised if `@stanfordbdhg/engagehf-models` already contained this validator. Not sure though.